.github/pull_request_template.md

```diff
@@ -1,10 +1,12 @@
 # Description
 
-> Detailed changes summary | testing | dependencies
+> Required; detailed changes summary | testing | dependencies
```

Comment:
Is this suppose to read "detailed changes summary" or "detailed changes" "summary | testing | dependencies" ?

Reply:
Any of these, just examples what to be expected. just that section is required

Reply:
Oh, yes this ends up rendered weird as is. Thoughts on actual comments?
<!--- loooooooooooooky a comment! --->
^ comment here ^